packages/button/src/Button.tsx

```diff
@@ -73,15 +75,20 @@ export const PureButton: React.FC<IButtonProps> = ({
         {...rest}>
         {!!prefix && (
           <Prefix
+            className={`${className} at-button__prefix`}
```

Comment:
i think that you don't need to add the same className for child components of Button, because for AT will be complicated to find unique button element

Reply:
it's actual for all components in current PR

Reply:
Agreed. 
I think - className should be optional for all components. We just need to check, if className provided from parent - so we add className to component. Also, I think prefix at-button or other prefixes should be provided within className (Developer will decide, which class name with prefix or not should be provided and why).
ClassNames should be flexible, and the reason, why we adding classNames for components shouldn't be implemented as hard-coded classNames.